src/ImageGroup.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { ReactNode } from 'react';
+
+export function ImageGroup({ children, cols = 2 }: { children: ReactNode; cols?: 1 | 2 }) {
+  return <div className={`not-prose grid sm:grid-cols-${cols} gap-4`}>{children}</div>;
+}
```

Comment:
```suggestion
import clsx from 'clsx';
export function ImageGroup({
  children,
  columnCount = 2,
  className,
}: {
  children: ReactNode;
  columnCount?: number;
  className?: string;
}) {
  return (
    <div className={clsx(`not-prose grid sm:grid-cols-${columnCount} gap-4`, className)}>
      {children}
    </div>
  );
}
```

Reply:
Sure. However, the prop signature will be inconsistent if we use `columnCount`. In [CardGroup, a similar prop is called `cols`](https://github.com/mintlify/components/blob/main/src/CardGroup.tsx#L3).